audio_service/lib/audio_service.dart

```diff
@@ -940,35 +946,83 @@ class AudioService {
     return handler;
   }
 
+  /// Allows to specify a URI for an album art (and optionally headers) that will be
+  /// used when [MediaItem.artUri] is null, or loading the art has failed.
+  ///
+  /// All schemes that are supported by [MediaItem.artUri] are supported here as well.
+  static Future<void> setFallbackArt(
+    Uri uri, {
+    Map<String, String>? headers,
+  }) async {
+    assert(_cacheManager != null, 'Call AudioService.init first');
+    _updateFallbackArtCache = true;
+    _fallbackArtUri = uri;
+    _fallbackArtHeaders = headers;
+    if (__handler != null) {
```

Comment:
Does this method really need to do anything other than set `_fallbackArt*` for the next time art is fetched in the normal course of things?

It's not ideal that this method needs to access `_handler` or `__handler`.

Reply:
Yes. This is needed to be able to dynamically update the fallback. In new PR I added a demonstration of this functionaly to `example_playlist.dart`.

Reply:
Yes. This is needed to be able to dynamically update the fallback. In new PR I added a demonstration of this functionality to `example_playlist.dart`.